specification/help/resource-manager/Microsoft.Help/preview/2023-01-01-preview/help.json

```diff
@@ -0,0 +1,614 @@
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Service specific error code which serves as the substatus for the HTTP error code.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "Service specific error type which serves as additional context for the error herein.",
+          "type": "string",
+          "readOnly": true
+        },
+        "message": {
+          "description": "Description of the error.",
+          "type": "string",
+          "readOnly": true
+        },
+        "details": {
+          "description": "An array of additional nested error response info objects, as described by this contract.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Error"
+          },
+          "x-ms-identifiers": [
+            "type",
+            "message"
+          ]
+        }
+      }
+    },
+    "DiscoveryResponse": {
+      "description": "Discovery response.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of solution metadata.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SolutionMetadata"
+          },
+          "x-ms-identifiers": [
+            "solutionId"
+          ]
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of solution metadata.",
+          "type": "string"
+        }
+      }
+    },
+    "SolutionMetadata": {
+      "description": "Diagnostic solution metadata.",
+      "type": "object",
+      "properties": {
+        "solutionId": {
+          "description": "Solution Id.",
+          "type": "string"
+        },
+        "solutionType": {
+          "description": "Solution Type.",
+          "type": "string"
+        },
+        "description": {
+          "description": "A detailed description of solution.",
+          "type": "string"
+        },
+        "requiredParameterSets": {
+          "description": "Required parameters for invoking this particular solution.",
+          "type": "array",
+          "items": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "x-ms-identifiers": []
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ScopeParameter": {
+      "name": "scope",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "This is an extension resource provider and only resource level extension is supported at the moment.",
+      "x-ms-parameter-location": "client",
+      "x-ms-skip-url-encoding": true
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version.",
+      "x-ms-parameter-location": "client"
+    },
```

Comment:
You can referene the ones from common types instead.

[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json#L666](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/types.json#L666)